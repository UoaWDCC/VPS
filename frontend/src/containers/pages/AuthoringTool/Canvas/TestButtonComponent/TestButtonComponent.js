import { Button } from "@material-ui/core";
import React from "react";

export default function TestButtonComponent({ id, selectElement }) {
  return (
    <Button
      className="btn top contained white"
      style={{ position: "absolute" }}
      color="default"
      variant="contained"
      id={id}
      onClick={selectElement}
    >
      TEST COMPONENT
    </Button>
  );
}
