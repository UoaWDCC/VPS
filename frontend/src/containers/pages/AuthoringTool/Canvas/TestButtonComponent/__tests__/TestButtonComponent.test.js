import React from "react";
import { render } from "@testing-library/react";
import TestButtonComponent from "../TestButtonComponent";

test("Scenario Selection page snapshot test", () => {
  const { baseElement } = render(<TestButtonComponent />);

  expect(baseElement).toMatchSnapshot();
});
