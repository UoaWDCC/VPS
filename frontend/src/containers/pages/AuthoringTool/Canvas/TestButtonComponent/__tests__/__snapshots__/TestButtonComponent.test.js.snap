// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scenario Selection page snapshot test 1`] = `
<body>
  <div>
    <div
      class="MuiBox-root MuiBox-root-2 btn top  white makeStyles-testButton-1"
    >
      <img
        alt="1"
        src="https://avatars.githubusercontent.com/u/52368549?v=4"
      />
    </div>
  </div>
</body>
`;
